tox.ini

```diff
@@ -0,0 +1,134 @@
+[tox]
+envlist = ci
+
+[testenv]
+whitelist_externals =
+    bash
+    coverage
+# shared directory for re-used packages
+envdir = {toxinidir}/.env
+passenv =
+    CI
+    CONDA_PREFIX
+    HOME
+    GITHUB_*
+    API_KEY_EIA
+    GOOGLE_APPLICATION_CREDENTIALS
+    SQLALCHEMY_WARN_20
+    PUDL_RMI_*
+setenv =
+    PUDL_RMI_INPUTS_DIR = {toxinidir}/inputs
+    PUDL_RMI_OUTPUTS_DIR = {toxinidir}/outputs
```

Comment:
@cmgosnell here's where the `PUDL_RMI_INPUTS` environment variable is getting set.